src/main/kotlin/no/nav/familie/ks/sak/integrasjon/økonomi/utbetalingsoppdrag/UtbetalingsoppdragGenerator.kt

```diff
@@ -17,12 +24,58 @@ import no.nav.familie.ks.sak.kjerne.beregning.domene.TilkjentYtelse
 import no.nav.familie.ks.sak.kjerne.beregning.domene.YtelseType
 import no.nav.familie.ks.sak.kjerne.personident.Aktør
 import org.springframework.stereotype.Component
+import java.time.LocalDate
 import java.time.YearMonth
 
 const val FAGSYSTEM = "KS"
 
 @Component
 class UtbetalingsoppdragGenerator {
+    fun lagUtbetalingsoppdrag(
+        saksbehandlerId: String,
+        vedtak: Vedtak,
+        forrigeTilkjentYtelse: TilkjentYtelse?,
+        nyTilkjentYtelse: TilkjentYtelse,
+        sisteAndelPerKjede: Map<IdentOgType, AndelTilkjentYtelse>,
+        erSimulering: Boolean,
+    ): BeregnetUtbetalingsoppdragLongId {
+        return Utbetalingsgenerator().lagUtbetalingsoppdrag(
+            behandlingsinformasjon =
+                Behandlingsinformasjon(
+                    saksbehandlerId = saksbehandlerId,
+                    behandlingId = vedtak.behandling.id.toString(),
+                    eksternBehandlingId = vedtak.behandling.id,
+                    eksternFagsakId = vedtak.behandling.fagsak.id,
+                    fagsystem = FagsystemKS.KONTANTSTØTTE,
+                    personIdent = vedtak.behandling.fagsak.aktør.aktivFødselsnummer(),
+                    vedtaksdato = vedtak.vedtaksdato?.toLocalDate() ?: LocalDate.now(),
+                    opphørAlleKjederFra = null,
+                    utbetalesTil = vedtak.behandling.fagsak.aktør.aktivFødselsnummer(),
+                    // Ved simulering når migreringsdato er endret, skal vi opphøre fra den nye datoen og ikke fra første utbetaling per kjede.
+                    opphørKjederFraFørsteUtbetaling = erSimulering,
+                ),
+            forrigeAndeler = forrigeTilkjentYtelse?.tilAndelData() ?: emptyList(),
+            nyeAndeler = nyTilkjentYtelse.tilAndelData(),
+            sisteAndelPerKjede = sisteAndelPerKjede.mapValues { it.value.tilAndelDataLongId() },
+        )
+    }
+
+    private fun TilkjentYtelse.tilAndelData(): List<AndelDataLongId> =
```

Comment:
Småpirk, men tror det er greit å ikke blande inn `AndelData` hvis det ikke finnes en klasse som heter det
```suggestion
    private fun TilkjentYtelse.tilAndelDataLongId(): List<AndelDataLongId> =
```